src/Auth/AuthController.php

```diff
@@ -0,0 +1,347 @@
+<?php
+
+namespace Canvas\Auth;
+
+use Canvas\Models\Sessions;
+use Canvas\Models\UserLinkedSources;
+use Canvas\Auth\Models\Users;
+use Baka\Http\Api\BaseController;
+use Canvas\Auth\Auth;
+use Exception;
+use Phalcon\Http\Response;
+use Phalcon\Validation;
+use Phalcon\Validation\Validator\Confirmation;
+use Phalcon\Validation\Validator\Email as EmailValidator;
+use Phalcon\Validation\Validator\PresenceOf;
+use Phalcon\Validation\Validator\StringLength;
+
+abstract class AuthController extends BaseController
```

Comment:
@rwhite27 I would break this on a trait and maybe have each of the functoins login and signup on a specific trait since we know we use it on other locations